### PR TITLE
Fix HSTS test in Infosec curriculum

### LINKED ISSUE #17593: Glitch starter project with wrong value for Strict-Transport-Security: max-age
#### Describe your problem and - if possible - how to reproduce it
In [Information Security with HelmetJS - Ask Browsers to Access Your Site via HTTPS Only with helmet.hsts()](https://learn.freecodecamp.org/information-security-and-quality-assurance/information-security-with-helmetjs/ask-browsers-to-access-your-site-via-https-only-with-helmet-hsts) challange we have to add middleware forcing browsers to use HTTPS protocol. In the helmet [documentation](https://helmetjs.github.io/docs/hsts/) is clearly described we have to use seconds to set the time period but in the Glitch starter project the variable provided is in milliseconds. By providing the right value for 90 days (ie 7776000 seconds), the test is failing because the value is not set to 777600000.

#### Add a Link to the page with the problem
https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-infosec/

#### Tell us about your browser and operating system

* Browser Name: Chrome
* Browser Version: 66
* Operating System: Win 10

#### If possible, add a screenshot here
image

### PR DESCRIPTION
**Pull request by Wocanilo**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #17593

The max-age attribute must be seconds, not miliseconds [1]

[1] https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security

### CODE REVIEW

**Comment by RandellDawson** [comment]
@Wocanilo It would also seem that https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-infosec/ would need to be updated?

**Comment by Dharmin2108** [comment]
.

**Comment by Dharmin2108** [comment]
.

**Comment by Dharmin2108** [comment]
.

**Comment by Dharmin2108** [comment]
.

**Comment by Dharmin2108** [comment]
/

**Comment by Dharmin2108** [comment]
/

**Comment by Dharmin2108** [comment]
.

**Comment by Dharmin2108** [comment]
'

**Comment by Dharmin2108** [comment]
5

**Comment by madurachanna** [comment]
This has to updated

**Comment by madurachanna** [comment]
It has to updated

**Comment by madurachanna** [comment]
It has to be updated

**Comment by madurachanna** [comment]
It has to updated

**Comment by madurachanna** [comment]
It has to updated

**Comment by Wocanilo** [comment]
@RandellDawson True, I just made a pull request with the changes.

**Comment by moT01** [comment]
The glitch project would need to be updated as well? Not sure if the projects are pulled from the boilerplates or what. But yes, this change seem fine to me, but we probably want to change all the locations of this problem at the same time. Additionally - projects that have been completed in the past might not pass anymore - not sure if there's anything we can do about that.

**Comment by ojeytonwilliams** [comment]
@moT01 Yeah, that's right, the glitch projects are built from the source on github, so as soon as the boilerplate is updated all new projects will use the updated code.  So, as you say, this will need to be synchronised.  Once we know that master is going live we can merge this and then merge the changes to the boilerplate as soon as this PR actually makes it to production.

@RandellDawson Do you know what we're doing with translations?  Still merging them?

File: curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/ask-browsers-to-access-your-site-via-https-only-with-helmet.hsts.english.md
Comment (ojeytonwilliams) [inline curriculum/challenges/english/06-information-security-and-quality-assurance/information-security-with-helmetjs/ask-browsers-to-access-your-site-via-https-only-with-helmet.hsts.english.md]:
```suggestion
  - text: maxAge should be equal to 7776000 s (90 days)
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
I'm not sure what, if anything, should be done about the other languages.  Anyways, I noticed one more conversion that was missed for the English.

**Comment by RandellDawson** [comment]
@ojeytonwilliams We don't have to do anything for the other languages, because they will never be implemented into production.

**Comment by ojeytonwilliams** [comment]
Thanks @RandellDawson.  Just that one remaining milli, then.

**Comment by RandellDawson** [comment]
@Wocanilo Can you please make the last change @ojeytonwilliams suggested, so we can get this PR merged.  Unfortunately, you did not allow maintainers to make changes on your behalf, so we can not make the change for you.

Thank you.

**Comment by ojeytonwilliams** [comment]
If @Wocanilo isn't available to address the review suggestions, we can just merge as is and then fix with another PR.

**Comment by Wocanilo** [comment]
I just accepted the last change.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
@ojeytonwilliams Just need your approval.

**Review by ojeytonwilliams — APPROVED** [review]

**Comment by ojeytonwilliams** [comment]
Sorry about the delay., this PR looks good now! My only concern is that the boilerplate will need changing as soon as master goes live.  Any ideas how best to do that?

**Comment by RandellDawson** [comment]
@raisedadead What is the best way to implement what @ojeytonwilliams is asking?

**Comment by raisedadead** [comment]
> Sorry about the delay., this PR looks good now! My only concern is that the boilerplate will need changing as soon as master goes live. Any ideas how best to do that?

We need to create a PR on the boilerplate-repo. Then on we wait for master to ship out and land this PR and the boilerplate PR all at the same time. It has to be orchestrated. 

The other option is to land the changes on the boilerplate, but not deploy it (someone actually goes into the Glitch account to pull the changes) untill master is shipped.

I am OK with either.

**Comment by ojeytonwilliams** [comment]
@Wocanilo made https://github.com/freeCodeCamp/boilerplate-infosec/pull/5, so I think all we need is to do is set up a reminder for when master goes live.

**Comment by RandellDawson** [comment]
@Wocanilo, @ojeytonwilliams, @raisedadead Now that `master` has been released to production, it is time to get the merge conflicts resolved and get this PR merged along with making sure the boilerplate PR is merged too.

**Comment by raisedadead** [comment]
Because the OP deleted their fork, this will now land via https://github.com/freeCodeCamp/freeCodeCamp/pull/37469

**Pull request closed without merging**
